packages/flutter/lib/src/cupertino/toggleable.dart

```diff
@@ -0,0 +1,252 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+/// A mixin for [StatefulWidget]s that implement iOS-themed toggleable
+/// controls (e.g.[Checkbox]es).
```

Comment:
```suggestion
/// controls (e.g. [CupertinoCheckbox]es).
```

Or describe how this class relates to the Material `Checkbox`.